src/main/resources/mybatis/timeSeries.xml

```diff
@@ -31,19 +45,31 @@
                                                      || 'collections/' || #{collectionId} || '/items/'
                                                      || ml.monitoring_location_identifier
                                                      || '/observations/statistical-time-series/' || time_series_unique_id)
+                                                     || case
+                                                           when best_time_series.best_ts_id is null then
+                                                                jsonb_build_object('best', null)
+                                                           when best_time_series.best_ts_id = time_series_unique_id and
+                                                                best_time_series.parameter_cd = observered_property_id and
+                                                                best_time_series.statistic_cd = statistic_id then
+                                                                   jsonb_build_object('best', true)
+                                                            else
+                                                               jsonb_build_object('best', false)
+                                                        end
```

Comment:
This is really hard to read here - it looks like the 'best' flag is an object that is tacked onto the end of the url.

Can 'best' just be added as an attribute of the same object that owns the url so that it is easier to see that it is an attribute child of it?

Also, do we want `best: null` and `best: false` in the document?  Another option would be to only include `best: true` and skip the others.  I think I prefer to make it an optional attribute to keep it short:  Just include `true` for the TS that is best, however, that may make it difficult/impossible to reformat as an attribute - I guess I would prefer the code readability here.

Reply:
The require from the ticket is to have best: null
push a refactor